examples/to_svg.py

```diff
@@ -122,3 +123,8 @@
 
     svg = viewer.to_svg()
     # svg = viewer.to_svg(file='viewer.svg')
+
+    # take screenshot
+    screenshot = viewer.screenshot()
```

Comment:
not sure if this belongs under something titled `to_svg`... maybe rename it or make this its own example?

Reply:
make new example, thanks